protocol-tests/src/main/java/fixtures/constantandclientparam/ConstantAndClientParameterServiceClientBuilder.java

```diff
@@ -0,0 +1,401 @@
+     *
+     * @param queryRequiredClientParam the queryRequiredClientParam value.
+     * @return the ConstantAndClientParameterServiceClientBuilder.
+     */
+    @Generated
+    public ConstantAndClientParameterServiceClientBuilder queryRequiredClientParam(int queryRequiredClientParam) {
+        this.queryRequiredClientParam = queryRequiredClientParam;
+        return this;
+    }
+
+    /*
+     * Constant query property on the client that is a required parameter
+     */
+    @Generated private int queryRequiredClientConstant = 100;
+
+    /**
+     * Sets Constant query property on the client that is a required parameter.
+     *
+     * @param queryRequiredClientConstant the queryRequiredClientConstant value.
+     * @return the ConstantAndClientParameterServiceClientBuilder.
+     */
+    @Generated
+    public ConstantAndClientParameterServiceClientBuilder queryRequiredClientConstant(int queryRequiredClientConstant) {
+        this.queryRequiredClientConstant = queryRequiredClientConstant;
+        return this;
+    }
+
+    /*
+     * Constant query property on the client that is a non-required parameter
+     */
+    @Generated private int queryNonRequiredClientConstant = 100;
+
+    /**
+     * Sets Constant query property on the client that is a non-required parameter.
+     *
+     * @param queryNonRequiredClientConstant the queryNonRequiredClientConstant value.
+     * @return the ConstantAndClientParameterServiceClientBuilder.
+     */
+    @Generated
+    public ConstantAndClientParameterServiceClientBuilder queryNonRequiredClientConstant(
+            int queryNonRequiredClientConstant) {
+        this.queryNonRequiredClientConstant = queryNonRequiredClientConstant;
+        return this;
+    }
+
+    /*
+     * Constant query property that is a required parameter
+     */
+    @Generated private int queryRequiredConstant = 100;
+
+    /**
+     * Sets Constant query property that is a required parameter.
+     *
+     * @param queryRequiredConstant the queryRequiredConstant value.
+     * @return the ConstantAndClientParameterServiceClientBuilder.
+     */
+    @Generated
+    public ConstantAndClientParameterServiceClientBuilder queryRequiredConstant(int queryRequiredConstant) {
+        this.queryRequiredConstant = queryRequiredConstant;
+        return this;
+    }
+
+    /*
+     * Constant query property that is a non-required parameter
+     */
+    @Generated private int queryNonRequiredConstant = 100;
+
+    /**
+     * Sets Constant query property that is a non-required parameter.
+     *
+     * @param queryNonRequiredConstant the queryNonRequiredConstant value.
+     * @return the ConstantAndClientParameterServiceClientBuilder.
+     */
+    @Generated
+    public ConstantAndClientParameterServiceClientBuilder queryNonRequiredConstant(int queryNonRequiredConstant) {
+        this.queryNonRequiredConstant = queryNonRequiredConstant;
+        return this;
+    }
```

Comment:
Why does it get to the client/builder? Could you double check your swagger and m4 output?

It is constant, not "from client".